contracts/contracts/lib/LibStaking.sol

```diff
@@ -87,8 +88,6 @@ library LibStakingReleaseQueue {
             delete self.releases[validator];
         }
 
-        payable(validator).sendValue(amount);
```

Comment:
This comment is repeated four times, answering here once only.

Do you have a reason why you prefer release done here? Does that mean LibStaking should also handle accept funds?

To release in this method, one has to pass collateralSource into this function then?

I just think `LibStaking` should just be bookkeeping and that's it. Fund handling is handled by caller, because:
1. fund transfer can be aggregated from other parts of the codebase. 
2. even if we confirmed it, we might not transfer immediately.

Reply:
2 things here. If you want to use the LibStaking just for book keeping that's fine. But I would rename the functions at least. When the function is called 'withdrawWithConfirm' but you are still expected to the to transfer it is a bit confusing. Given that now thew transfer was included.

Reply:
any names you have in mind, actually I thought the current one is not bad, haha.